> Wait on https://www.githubstatus.com/incidents/k5z4d1v1tqmt to resolve. This change is a no-op, both variables hold the same value.

Run node ./tools/ci-tools.ts set-pr-labels
GitHub API POST https://api.github.com/repos/go-gitea/gitea/issues/37827/labels failed (403): {"message":"Resource not accessible by integration","documentation_url":"https://docs.github.com/rest/issues/labels#add-labels-to-an-issue","status":"403"}
Error: Process completed with exit code 1.

--- a/.github/workflows/pull-labeler.yml
+++ b/.github/workflows/pull-labeler.yml
@@ -27,7 +27,7 @@ jobs:
     timeout-minutes: 5
     permissions:
       contents: read
-      issues: write
+      pull-requests: write
     steps:
       # Base-branch checkout only: pull_request_target runs with elevated token; never run PR-head code here.
       - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
@@ -44,4 +44,4 @@ jobs:
         env:
           PR_TITLE: ${{ github.event.pull_request.title }}
           PR_NUMBER: ${{ github.event.pull_request.number }}
-          GITHUB_TOKEN: ${{ secrets.GITEABOT_TOKEN || github.token }}
+          GITHUB_TOKEN: ${{ github.token }}

